gtest-parallel

```diff
@@ -447,9 +449,9 @@ def worker():
     job = None
     test_lock.acquire()
     if job_id < len(tests):
-      (_, test_binary, test, command) = tests[job_id]
+      (_, test_binary, test, command), test_index = tests[job_id]
```

Comment:
Preserve the outer parentheses?
`((_, test_binary, test, command), test_index) = tests[job_id]`